docs/buildah-config.md

```diff
@@ -72,6 +72,10 @@ The shell instruction allows the default shell used for the shell form of comman
 
 Note: this setting is not present in the OCIv1 image format, so it is discarded when writing images using OCIv1 formats.
 
+**--stop-signal** *signal*
+
+Specifies a *signal* and the system call this *signal* that will be sent to the container to exit.
```

Comment:
Suggest: "Sets the system call *signal* that will be sent to the container to exit."

Reply:
I would explain it further as I did above.



set default stop signal for container. This signal will be sent when container is stopped, default is SIGINT.